src/index.html

```diff
@@ -3,10 +3,336 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap"
+      rel="stylesheet"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="index.html" class="header__logo_link">
+        <img
+          src="images/logo.png"
+          alt="MOYO-logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list_item">
+            <a href="#" class="is-active">Apple</a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Laptops &#38; computers
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main_section">
+      <div class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="imac image"
+            class="card__image"
+          >
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
```

Comment:
There can be only one `h1` on the page, because it's main heading, so it can't be in a cards, `h2` or `h3` will be better here